processor/README.md

```diff
@@ -187,16 +201,147 @@ These parameters are already provided by the cart created in composable commerce
 - merchantReturnUrl:
 
 ### Confirm payment
-TBC 
+Submits details for a payment to Adyen platform to confirm a payment. It is only necessary when the payment is initialized through [create payment](#create-payment)
+#### Endpoint
+`POST /payments/details`
+
+#### Request Parameters
+The request body is same as [adyen checkout create payment details request](The request body is same as [adyen checkout create payment request](https://docs.adyen.com/api-explorer/Checkout/71/post/payments#request)) with following parameters :
+- authenticationData: Data for 3DS authentication.
+- details: A collection of result returned from the `/payments` call.
+- paymentData: Encoded payment data returned from the `/payments` call. If `AuthenticationNotRequired` is received as `resultCode` in the `/payments` response, use the `threeDSPaymentData` from the same response. If the `resultCode` is `AuthenticationFinished`, use the `action.paymentData` from the same response.
+
+#### Response Parameters
+It returns following attributes in response
+- 
+- 
+- paymentReference: Unique identifier of payment resources updated in commercetools composable commerce.
+-  
 
 ### Get supported payment components
-TBC
+Private endpoint protected by JSON Web Token that exposes the payment methods supported by the connector so that checkout application can retrieve the available payment components. 
+#### Endpoint
+`GET /operations/payment-components`
 
-### Get status
-TBC
+#### Request Parameters
+N/A
+
+#### Response Parameters
+Now the connector supports payment methods such as `card`, `iDEAL`, `PayPal` and `Sofort`
+```
+{
+    components: [
+        {
+          type: 'card',
+        },
+        {
+          type: 'ideal',
+        },
+        {
+          type: 'paypal',
+        },
+        {
+          type: 'sofort',
+        },
```

Comment:
Sofort is not supported anymore

Reply:
https://github.com/commercetools/connect-payment-integration-adyen/pull/104/commits/5299dd2354b51e7b3167a1d3b70e5670df77803a